inyoka_theme_default/templates/base.html

```diff
@@ -12,18 +12,21 @@
 
 <!DOCTYPE html>
 <html>
-{% block head %}
+
 <head>
+{% block head %}
+  {% block head_meta %}
```

Comment:
maybe name the block only meta or meta-informations

that meta is in head is no information which is needed, because there is no other valid place for it.